python/lsst/ts/genericcamera/genericcameracsc.py

```diff
@@ -953,15 +1000,22 @@ async def configure(self, config):
                 f"No config found for sal_index={self.salinfo.index}"
             )
 
+        if config.s3instance != "none":
```

Comment:
If you agree to make the change I suggested in the configuration schema, you can actually make `config.s3instance = None`, in which case, this could be 

```
if config.s3instance is None:
```

Though, I think it should actually be:

```
self.use_lfa = config.s3instance is None
```

Reply:
Should be `is not None`, but went with that syntax.